### ISSUE TITLE
Is it possible to use @focus and @mouseover in dynamic macros?

### ISSUE THREAD

**Issue by korv114** [Verubato/framesort#14]
If not, that'd be a pretty useful addition to make this feature perfect :)

**Comment by Verubato** [comment]
Yep you can do that, here's an example:

```lua
#FrameSort mouseover, focus
/cast [@mouseover,help][mod:shift,@focus][] Spell
```

**Comment by korv114** [comment]
I actually tried it like that but it was replacing them with @none so I assumed it wouldn't work. And didn't seem to work outside arena at least

**Comment by Verubato** [comment]
Ah snap, there is a bug where it won't work outside of a group.
I'll push a fix for it and release.

**Comment by Verubato** [comment]
Issue fixed in release 5.2.1.

@korv114 if you update FrameSort it should be fixed now.
Thanks for notifying me about this!

**Comment by korv114** [comment]
Awesome, thanks!
